modules/install-sno-requirements-for-installing-on-a-single-node.adoc

```diff
@@ -9,15 +9,15 @@ Installing {product-title} on a single node alleviates some of the requirements
 
 * *Administration host:* You must have a computer to prepare the ISO, to create the USB boot drive, and to monitor the installation.
 
-* *Bare metal installation:* Installing {product-title} on a single node on bare metal requires that you specify the `platform.none: {}` parameter in the `install-config.yaml` configuration file.
+* *Supported platforms:* Installing {product-title} on a single node is supported on bare metal, vSphere, Red Hat OpenStack, and Red Hat Virtualization platforms. In all cases the `platform.none: {}` parameter must be specified in the `install-config.yaml` configuration file.
```

Comment:
Consider, 
"In all cases, you must specify the `platform.none: {}` parameter in the `install-config.yaml` configuration file." 
to avoid passive voice.

Reply:
Updated as suggested.